### PR TITLE
Update set-up-attributes.md

### PR DESCRIPTION
**Pull request by JudyWeiner**
Added update on how to add hierarchies and derived attributes post initial setup.

### CODE REVIEW

**Comment by scanum** [comment]
Hi @tiburd hope all is well. Could you please help with this PR? Thanks!

File: Viva/glint/setup/set-up-attributes.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Select your attributes from the dropdown menu for each hierarchy group.
 - To delete a group or level, select the **trash can** symbol.
 
 > [!NOTE]
-> Hierarchy groups cannot be edited after setup. Confirm that all groups and hierarchies contain accurate attributes before saving.
+> Hierarchy groups can only be set up during initial schema set up. To add hierarchies at the time of editing the schema, [reach out to support](https://learn.microsoft.com/en-us/microsoft-365).
```
Comment (JohanFreelancer9) [inline Viva/glint/setup/set-up-attributes.md:78]:
```suggestion
> Hierarchy groups can only be set up during initial schema setup. To add hierarchies at the time of editing the schema, [contact Microsoft Support](https://support.microsoft.com/contactus).
```

File: Viva/glint/setup/set-up-attributes.md [hunk 2]
```diff
@@ -85,6 +85,9 @@ Viva Glint calculates attributes based on data sent in your file. In this sectio
 2. Select the attribute from the dropdown menu used to calculate each derived attribute. For example, Manager ID, Hire Date, and Birth Year.
 3. Select **Continue**.
 
+   > [!NOTE]
+> Derived groups can only be set up during initial schema set up. To add hierarchies at the time of editing the schema, [reach out to support](https://learn.microsoft.com/en-us/microsoft-365).
```
Comment (JohanFreelancer9) [inline Viva/glint/setup/set-up-attributes.md:89]:
```suggestion
> Derived groups can only be set up during initial schema setup. To add hierarchies at the time of editing the schema, [Contact Microsoft Support](https://support.microsoft.com/contactus).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JudyWeiner, kindly commit the suggested changes. Many thanks.

**Comment by JudyWeiner** [comment]
@johanFreelancer9
I hope I did this correctly! Also not sure how to tag you.
Thanks for your help!

Judy Weiner
Viva Glint Content Manager
Microsoft Viva
Mobile: 248-921-7657
***@***.***

[signature_351660960]

From: JohanFreelancer9 ***@***.***>
Date: Wednesday, August 16, 2023 at 3:50 AM
To: MicrosoftDocs/Viva ***@***.***>
Cc: Judy Weiner ***@***.***>, Mention ***@***.***>
Subject: Re: [MicrosoftDocs/Viva] Update set-up-attributes.md (PR #660)

@JohanFreelancer9 requested changes on this pull request.

Copy review - @JudyWeiner<https://github.com/JudyWeiner>, kindly commit the suggested changes. Many thanks.

**Comment by JohanFreelancer9** [comment]
@JudyWeiner. Hi Judy. You should see a "Commit" button next to each of my suggested changes. Just select {click) these two buttons in turn and you are done. Also, once you have done that I will be tagged automatically. Thanks so much!

**Comment by scanum** [comment]
We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Comment by JohanFreelancer9** [comment]
@scanum, the OP has in fact committed these suggestions, so can we please re-open the PR so that it can be merged? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
Thanks @JohanFreelancer9 for checking! 

Adding @get-itips for review.

**Comment by scanum** [comment]
@MelissaBarry 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@traya1 Thomas can you please help on the merge conflicts here? Thanks

**Comment by traya1** [comment]
Hi Team,

I will be meeting with Judy tomorrow and will discuss this PR with her then. Most likely the outcome will be closing this PR and moving the changes to the private repo.

FYI: @tiburd & @JudyWeiner

**Comment by traya1** [comment]
The article was deleted and redirected. So we have closed the PR.

**Pull request closed without merging**